blueprints/app/files/package.json

```
@@ -26,9 +26,9 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.7<% if (!typescript) { %>",
+    "@babel/core": "^7.26.7",
     "@babel/eslint-parser": "^7.26.5",
-    "@babel/plugin-proposal-decorators": "^7.25.9<% } %><% if (typescript) { %>",
+    "@babel/plugin-proposal-decorators": "^7.25.9<% if (typescript && emberData) { %>",
```

The emberData check isn't needed on 6.2 and seems to be the reason for the CI faillure.
```suggestion
    "@babel/plugin-proposal-decorators": "^7.25.9<% if (typescript) { %>",
```